restore-sql-server-transaction-logs/scheduled-upload/settings.json

```diff
@@ -0,0 +1,10 @@
+{
+    "DateTimeFormat":  "yyyy-MM-dd HH:mm:ss.fff",
+    "BucketName":  "<your-bucket-name>",
+    "BucketUploadFolder":  "uploaded",
+    "GoogleAccountKeyFile":  "<full-path-to-the-key-file>",
+    "LocalPathForBackupFiles":  "<full-path-to-the-backup-files>",
+    "LastRunMaxWriteDateTimeUtc":  "",
+    "LastRunTimeUtc":  "",
+    "LastRunUploadedObjectsNumber":  0
```

Comment:
These look like data from past runs, not configuration. I suggest to move these to a dedicated file.

Reply:
Indeed, I added them to a separate log file.

Reply:
STGM